docs/developer-guide.md

```diff
@@ -8,16 +8,22 @@
 
 ## Build from source code
 
-Pull in the dependencies
+Check source code as follows:
 
 ```
-dep ensure --vendor-only -v
+make check
+```
+
+If there are some errors for go fmt, uodate the go fmt as follows:
+
+```
+make update
 ```
 
 You can build all images from source.
 
 ```bash
-./scripts/v1alpha1/build.sh
+make build
```

Comment:
can you refactor it to add version arg to `bash scripts/v1alpha1/build.sh` or `bash scripts/v1alpha2/build.sh`

Reply:
My thinking is as follows, I will try to update `makefile` when `v1alpha2` become stable by making `make build` to build `v1alpha2` and introduce another `make build-v1alpha1` to build the old `v1alpha1` version.

But for now, I want to use `make build` default to `v1alpha1` as `v1alpha2` is not ready yet, comments?

Reply:
SGTM